.github/workflows/push_latest.yml

```
@@ -206,7 +208,7 @@ jobs:
           | while read file; do process_tags "$file"; done
         }
         echo "TAGS_MARKDOWN<<EOF" >> $GITHUB_ENV
-        echo "## Recent Tags" >> $GITHUB_ENV
+        echo "## Recent tags and respective `Dockerfile` links" >> $GITHUB_ENV
```

My feeling is to revert this line since there's nothing here to advertise

```suggestion
        echo "## Recent Tags" >> $GITHUB_ENV
```

---

Either way I think it's good enough as-is. No objections from my side.